api/specs.proto

```diff
@@ -91,6 +91,7 @@ message ServiceSpec {
 // ReplicatedService sets the reconciliation target to certain number of replicas.
 message ReplicatedService {
 	uint64 replicas = 1;
+	repeated string task_remove = 2;
```

Comment:
I am not sure if letting users delete tasks is something we want to expose directly. However, if we do want to expose it, this isn't the right way. `ReplicatedService` is part of a service's declarative state, which shouldn't reference individual tasks. Instead of updating the service and filling in a `task_remove` field, the CLI could simply invoke the `RemoveTask` RPC.